torch/_jit_internal.py

```diff
@@ -33,6 +33,19 @@
 boolean_dispatched: 'weakref.WeakKeyDictionary[Callable, Dict[str, Callable]]' = weakref.WeakKeyDictionary()  # noqa: T484
 
 
+class FrontendError(Exception):
```

Comment:
FYI: I moved this in order to get the source range to render properly and consistently.